docs/user_guide/hopsworks/jobs.rst

```diff
@@ -93,7 +100,40 @@ Additionally, users click on a job and view additional information about their r
 
 .. _jobs-ui-exec-logs.png: ../../_images/jobs-ui-exec-logs.png
 .. figure:: ../../imgs/jobs-ui-exec-logs.png
-    :alt: Job run logs
+    :alt: Job logs
     :target: `jobs-ui-exec-logs.png`_
     :align: center
-    :figclass: align-center
\ No newline at end of file
+    :figclass: align-center
+
+    Job aggregated logs
+
+Python
+------
+
+There are three ways of running Python programs in Hopsworks:
+
+* Jupyter notebooks: Covered in the Jupyter section of the user guide.
+* Jobs UI
+* Programmatically
+
+The GIF below demonstrates how to create a Python job from the **Jobs UI** by selecting a python file that is already
+uploaded in a Hopsworks dataset and attaching a few other files to be immediately available to the application at
+runtime. However, any file can be made available to the application at runtime by using in the Python app to run, the
+``copy_to_local`` function of the ``hdfs`` module of the ``hops`` Python library
+http://hops-py.logicalclocks.com/hops.html#module-hops.hdfs
+
+.. _python-new-job.gif: ../../_images/python-new-job.gif
+.. figure:: ../../imgs/python-new-job.gif
+    :alt: Python new job UI
+    :target: `python-new-job.gif`_
+    :align: center
+    :figclass: align-center
+
+    Create a new Python job from the Jobs UI
+
+The executable Python program does not need to be uploaded from the UI, it can also be done so by using in the Python app to run, the
```

Comment:
"can be done so by using in the Python app to run"
This doesn't really parse or make sense.

Reply:
thanks, fixed